From 8e6341b0bce009ab5f46cd55f5f1598a9408051b Mon Sep 17 00:00:00 2001
From: Jean-Yves Avenard <jyavenard@mythtv.org>
Date: Tue, 15 Jul 2014 15:48:11 +1000
Subject: [PATCH] metadata: Fix GetGrabber

It would never have used the grabber set in the inetref

(cherry picked from commit 61a511562c00858ffb767a24ac9fd586cdf83fe2)
---
 mythtv/libs/libmythmetadata/metadatagrabber.cpp |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mythtv/libs/libmythmetadata/metadatagrabber.cpp b/mythtv/libs/libmythmetadata/metadatagrabber.cpp
index 74e5328..ff5a304 100644
--- a/mythtv/libs/libmythmetadata/metadatagrabber.cpp
+++ b/mythtv/libs/libmythmetadata/metadatagrabber.cpp
@@ -162,7 +162,7 @@ MetaGrabberScript MetaGrabberScript::GetGrabber(GrabberType defaultType,
         lookup->GetInetref() != "00000000")
     {
         // inetref is defined, see if we have a pre-defined grabber
-        MetaGrabberScript grabber = lookup->GetInetref();
+        MetaGrabberScript grabber = FromInetref(lookup->GetInetref());
 
         if (grabber.IsValid())
         {
-- 
1.7.10.2

